proarc-common/src/main/java/cz/cas/lib/proarc/common/export/mets/structure/MetsElementVisitor.java

```diff
@@ -199,6 +199,29 @@ protected void initHeader(IMetsElement metsElement) throws MetsExportException {
     }
 
     /**
+     * Returns the name of title if element is issue
+     */
+    private String getTitle(IMetsElement metsElement) throws MetsExportException {
+        if (isIssue(metsElement)) {
+            Node partNode = MetsUtils.xPathEvaluateNode(metsElement.getModsStream(), "//*[local-name()='mods']/*[local-name()='titleInfo']/*[local-name()='title']");
+            if (partNode == null){
+                throw new MetsExportException("Error - missing title. Please insert title.");
+            }
+            return partNode.getTextContent() + " ";
+        }
+        return "";
+    }
+
+    /**
+     * Returns true if element is issue, else return false
+     */
+    public boolean isIssue(IMetsElement metsElement) throws MetsExportException {
```

Comment:
Týká se to fakt jenom úrovně issue? Co například ročník? To issue (githubovské) je trochu vágní. Máš to v souladu s NDK speckou?